spec/services/quota_search_service_spec.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable RSpec/MultipleMemoizedHelpers
```

Comment:
I appreciate this isn't your code but lots of memoized helpers is a bit of a code smell - I think either we should be deciding to ignore this for now (have have lint fail - allowing the next developer to revisit it) or fixing the issue.

Is it worth adding either extending the factories to allow easily building the desired scenario, or just adding a `shared_context` block to build up the desired data?

Reply:
yeah, all those lets make the code a bit smelly, I'll see how we can reduce those.

Reply:
Maybe do that in a separate PR @alexdesi and we can just accept that the linting doesn't pass for this file for now

Reply:
Since it is not a fast thing, yes it's better to keep it disabled for now, so Madhu can start testing this on Monday